vueui/src/components/view/cooking/microwave.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div>
+    <template v-if="on == true">
+      <p>Cooking...</p>
+      <vui-progress v-if="cook_time" :value="cur_time" :max="cook_time"/>
+      <vui-button class="danger danger-control" icon="exclamation-triangle" push-state :params="{abort: 1}">Abort!</vui-button>
```

Comment:
`push-state` is absolutely unnecessary here, as you do not read client side data or modify it on client side. This just adds bloat to network traffic.